pkg/network/protocols/tls/java/hotspot.go

```diff
@@ -189,7 +189,7 @@ func (h *Hotspot) connect(withCredential bool) (close func(), err error) {
 		return nil, err
 	}
 
-	if err := conn.SetDeadline(time.Now().Add(3 * time.Second)); err != nil {
+	if err := conn.SetDeadline(time.Now().Add(30 * time.Second)); err != nil {
```

Comment:
that sounds a bit large, this code path is called for each java detected java process in a separate go routine right? (`connect` function is an internal method called from `newJavaProcess` which is used as a callback when subscribing for processMonitor on `exec` events)

Reply:
removed that change for now